pages/desktop/home.py

```diff
@@ -271,21 +273,34 @@ def click_secondary_module_link(self):
                 target = link.get_attribute('target')
                 # external links are opened in new tabs, so we need to account for multiple windows
                 if target == '_blank':
-                    home_tab = self.selenium.current_window_handle
                     link.click()
                     self.wait.until(EC.number_of_windows_to_be(2))
                     new_tab = self.selenium.window_handles[1]
                     self.selenium.switch_to_window(new_tab)
-                    # waiting for an element in the new page to be loaded
+                    # editorial might change these links when they need to push new content and we don't know
+                    # in advance what that content might be; also, we want to avoid frequent maintenance for
+                    # these tests; the solution used is to verify that the content we link to is available
+                    # (i.e. we check that the page response status is 200)
+                    self.wait.until(custom_waits.url_not_contins('about:blank'))
+                    page = requests.head(self.selenium.current_url)
+                    try:
+                        assert page.status_code == 200
+                    except AssertionError:
+                        print(f'The response status code was {page.status_code}')
```

Comment:
You don't need a try/except block, you can add a message to the assert.
```suggestion
                    assert page.status_code == 200, f"the response code was {page.status_code}, not 200".
```

Something like that.

Reply:
Yes, I like that. I will use this format more in the future.